database/seeders/InviteKeysTableSeeder.php

```diff
@@ -64,6 +64,7 @@ public function run()
             'role' => Roles::Police,
         ]);
 
+        // This one is not in use
```

Comment:
Waarom staat het er dan?

Geld ook voor de rest.

Reply:
Dit geeft aan welke invite-keys als default in gebruik zijn door de seerder. De comments helpen om snel te zien welke codes je kan gebruiken om te testen.